docs/operation-guide.md

```diff
@@ -62,15 +62,28 @@ $ kubectl get svc -n ${namespace} # check the available services
 
 ## Scale TiDB cluster
 
-TiDB Operator has full support of horizontal scaling. But for vertical scaling, if you're using local volumes for PD and TiKV, then scaling up may cause pod pending if the node doesn't have enough resources. So it's not recommended to do vertical scaling.
+TiDB Operator supports both horizontal and vertical scaling, but there are some caveats for storage vertical scaling.
+
+* Kubernetes is v1.11 or later, please reference [the official blog](https://kubernetes.io/blog/2018/07/12/resizing-persistent-volumes-using-kubernetes/)
+* Backend storage class supports resizing. (Currently only a limited of network storage class supports resizing)
+
+When using local persistent volumes, even CPU and memory vertical scaling can cause problems because there may be not enough resources on the node.
+
+Due to the above reasons, it's recommended to do horizontal scaling other than vertical scaling when workload increases.
+
+### Horizontal scaling
 
 To scale in/out TiDB cluster, just modify the `replicas` of PD, TiKV and TiDB in `values.yaml` file. And then run the following command:
 
 ```shell
 $ helm upgrade ${releaseName} charts/tidb-cluster
 ```
 
-To scale up/down TiDB cluster, modify the cpu/memory limits and requests of PD, TiKV and TiDB in `values.yaml` file. And then run the same command as above. (Note: This may fail when using local volumes.)
+### Vertical scaling
+
+To scale up/down TiDB cluster, modify the cpu/memory/storage limits and requests of PD, TiKV and TiDB in `values.yaml` file. And then run the same command as above.
```

Comment:
Can we add details about the TiKV config here?

Reply:
The `block-cache-size` parameters scattered in TiKV toml configuration has comment on how to set these values as pointed out in #35. The following **Note** already pointed to that issue.